### DISCUSSION TITLE
A simple ANSI color implementation and CPU protector for illegal instructions

### DISCUSSION THREAD

**Discussion by ghost** [joncampbell123/dosbox-x#5303] [Ideas]
The `color.com` simulating the `color` command in Windows. 
<img width="553" alt="20241207211842" src="https://github.com/user-attachments/assets/69bed9db-9532-4f5f-a1a2-1b7d94e70f24">
The `protect.com` automatically handles certain interrupts that will be called when CPU encounters errors.

<img width="353" alt="20241207211904" src="https://github.com/user-attachments/assets/842d6ea7-a608-499e-9e2c-f35fc0c59570">

Before installing CPU Protector:

<img width="541" alt="20241207212431" src="https://github.com/user-attachments/assets/6a563f7a-8311-4ac1-b1b7-d51e229383b2">

The source and builds can be found here, with full assembler environment:
[asm.zip](https://github.com/user-attachments/files/18048548/asm.zip)

**Comment by joncampbell123** [comment]
I assume this project will probably see development or at least bug fixes over time, correct?

I recommend you put this code in a git repository and point to the repository. Binaries can be uploaded as part of releases.

Sound good?
